docs/curl.1

```diff
@@ -2084,6 +2084,22 @@ that is older than the given date/time, default is a document that is newer
 than the specified date/time.
 
 If this option is used several times, the last one will be used.
+.IP "--tls-max <VERSION>"
+(SSL) Defines a range of supported TLS versions up to VERSION. A minimum is defined
+by arguments tlsv1.0 or tlsv1.1 or tlsv1.2.
+
+.RS
+.IP "default"
+Use up to recommended version.
+.IP "1.1"
+Use up to TLSv1.1 . The supported minimum is tlsv1.0.
+.IP "1.2"
+Use up to TLSv1.2 . The supported minimum is tlsv1.0 or tlsv1.1.
+.IP "1.3"
+Use up to TLSv1.3 . The supported minimum is tlsv1.0 or tlsv1.1 or tlsv1.2.
+.RE
+
+See also \fI--tlsv1.0\fP and \fI--tlsv1.1\fP and \fI--tlsv1.2\fP. \fI--tls-max\fP requires that the underlying libcurl was built to support TLS. Added in 7.53.0.
```

Comment:
docs/curl.1 is generated document by  command: `docs/cmdline-opts/gen.pl mainpage`